Core/HLE/sceNetResolver.cpp

```diff
@@ -58,30 +58,39 @@
 #define INADDR_NONE 0xFFFFFFFF
 #endif
 
+static int mCurrentNetResolverId = 1;
+static std::unordered_map<u32, std::shared_ptr<NetResolver>> mNetResolvers;
+static std::shared_mutex mNetResolversLock;
+
+// NOTE: It starts as true, needed for Outrun 2006.
+static bool g_netResolverInitialized = true;
+
 static int sceNetResolverInit() {
-    g_Config.mHostToAlias["socomftb2.psp.online.scea.com"] = "67.222.156.250";
-    g_Config.mHostToAlias["socompsp-prod.muis.pdonline.scea.com"] = "67.222.156.250";
-    SceNetResolver::Init();
-    return hleLogSuccessInfoI(Log::sceNet, 0);
+	// TODO: Move this to infra-jns.config! This isn't ok.
```

Comment:
We probably want to keep the HostAliases feature for private server developers to override DNS from AutoDNS/server updated infra-dns.json, since they usually use LAN IP during early development.

And with PPSSPP's built-in DNS Resolver, overriding it through system's hosts file won't work anymore, so PPSSPP need to provide a way for them to use their LAN IP.

Reply:
The hostAliases feature will be kept, however these hardcoded lines right here will be moved to the json.